lnwallet/btcwallet/psbt.go

```diff
@@ -59,24 +63,45 @@ func (b *BtcWallet) FundPsbt(packet *psbt.Packet, minConfs int32,
 		keyScope   *waddrmgr.KeyScope
 		accountNum uint32
 	)
+
+	// No change key scope should have been specified for custom accounts.
```

Comment:
Would be nice if we could extend the `runFundAndSignPsbt` integration test (see `lntest/itest/lnd_psbt_test.go`) with another loop that tests both change output types and asserts the generated change address/pkScript.

Reply:
added a nested for loop + asserts!